spec_helper.rb

```diff
@@ -14,9 +14,20 @@ module SpecHelper
 unless ENV["DEBUG"]
   fastlane_tests_tmpdir = "#{Dir.tmpdir}/fastlane_tests"
   $stdout.puts("Changing stdout to #{fastlane_tests_tmpdir}, set `DEBUG` environment variable to print to stdout (e.g. when using `pry`)")
+  $orig_stdout = $stdout
```

Comment:
helper code. Could be moved in a different PR, or discarded,

Reply:
I can't figure out what this is doing 😅  Does this do something different than what setting `DEBUG` would do? 🤔

Reply:
@joshdholtz this was meant to be used together with the helper method below. It allows me to add code to re-activate the output for just one or few tests.

Reply:
I'll resolve this as well.